### PR TITLE
[Aftershock] strongman to vat grown bodyguard

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
SUMMARY: Mods "Obsolete Strongman and replace"

#### Purpose of change
Circus strongman didn't really fit the Aftershock direction.

#### Describe the solution
Changed to vatgrown bodyguard.

#### Describe alternatives you've considered
Obsoleting strongman.

#### Testing
TBD

### CODE REVIEW

File: data/mods/Aftershock/player/professions.json [hunk 1]
```diff
@@ -181,12 +181,16 @@
...
-    "description": "You were born with a genetic defect that made your body unnaturally strong.  Your life was strange and multifaceted, and ultimately, you ended up traveling with a troupe up and down the coastline, putting on shows of strength  (NOTE: To preserve balance, choose this profession *before* adjusting your stats.  Or don't.  I'm a description, not a cop.)",
-    "points": 2,
-    "traits": [ "AFS_PROF_STRONGMAN" ],
-    "items": { "both": [ "b_shorts", "undershirt" ], "male": [ "briefs" ], "female": [ "boy_shorts" ] }
+    "ident": "afs_vatgrown_bodyguard",
+    "name": "Vatgrown Bodyguard",
+    "description": "Grown in some corporate laboratory to provide the perfect bodyguard, with nothing to live for but your client.  The Cataclysm ignored social strata and somehow you outlived your master.  Now no one knows the conditioning words to force your obedience.  While the world burns, you get your first taste of life. (NOTE: To preserve balance, choose this profession *before* adjusting your stats.  Or don't.  I'm a description, not a cop.)",
+    "points": 3,
+    "traits": [ "AFS_PROF_STRONGMAN", "SLOWREADER", "LIGHTWEIGHT" ],
```
Comment (anothersimulacrum) [inline data/mods/Aftershock/player/professions.json]: You may need to adjust the `PROF_STRONGMAN` trait name/description.

Comment (Maleclypse) [reply data/mods/Aftershock/player/professions.json]: Good point. Thanks!

File: data/mods/Aftershock/player/professions.json [hunk 2]
```diff
@@ -181,12 +181,16 @@
...
-    "ident": "afs_circus_strongman",
-    "name": { "male": "Circus Strongman", "female": "Circus Strongwoman" },
-    "description": "You were born with a genetic defect that made your body unnaturally strong.  Your life was strange and multifaceted, and ultimately, you ended up traveling with a troupe up and down the coastline, putting on shows of strength  (NOTE: To preserve balance, choose this profession *before* adjusting your stats.  Or don't.  I'm a description, not a cop.)",
-    "points": 2,
-    "traits": [ "AFS_PROF_STRONGMAN" ],
-    "items": { "both": [ "b_shorts", "undershirt" ], "male": [ "briefs" ], "female": [ "boy_shorts" ] }
+    "ident": "afs_vatgrown_bodyguard",
+    "name": "Vatgrown Bodyguard",
+    "description": "Grown in some corporate laboratory to provide the perfect bodyguard, with nothing to live for but your client.  The Cataclysm ignored social strata and somehow you outlived your master.  Now no one knows the conditioning words to force your obedience.  While the world burns, you get your first taste of life. (NOTE: To preserve balance, choose this profession *before* adjusting your stats.  Or don't.  I'm a description, not a cop.)",
```
Comment (John-Candlebury) [inline data/mods/Aftershock/player/professions.json]:
```suggestion
    "description": "Grown in some corporate laboratory to provide the perfect bodyguard, with nothing to live for but your client.  The Cataclysm ignored social strata and somehow you outlived your master.  Now no one knows the conditioning words to force your obedience.  While the world burns, you get your first taste of life. ",
```
I think that breaking the 4th wall like this is bad . If you feel the balance of the profession still needs tweaking, you should maybe up its cost, or include more defects.

Comment (Maleclypse) [reply data/mods/Aftershock/player/professions.json]: I left the old warning from the original.  I'm happy with the upped cost and the added defects, I think.  I'm also not sure if what they are describing isn't fixed in current versions.

File: data/mods/Aftershock/items/weapons.json [hunk 3]
```diff
@@ -226,5 +226,23 @@
     "volume": "1750 ml",
     "bashing": 22,
     "price": 20000
+  },
+  {
+    "id": "ceramic_knife",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "bodyguard knife",
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/items/weapons.json]: Can you add a MA extension .json for this? There's an example in Magiclysm and Mythological weapons on how to do it.

Comment (Maleclypse) [reply data/mods/Aftershock/items/weapons.json]: Do I need to add that in a separate PR?  I'm not 100% what you are asking but I'll try and figure out.  Did I use an ID or something that is in another place?

File: data/mods/Aftershock/martialarts.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    "type": "martial_art",
+    "name": "Fencing",
+    "extend": { "weapons": [ "afs_energy_saber_on" ] }
+  },
+  {
+    "id": "style_medievalpole",
+    "copy-from": "style_medievalpole",
+    "type": "martial_art",
+    "name": "Fior Di Battaglia",
+    "extend": { "weapons": [ "afs_titanium_bat" ] }
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/martialarts.json]: There's no part of a bat's end that can be used to hook opponents

File: data/mods/Aftershock/martialarts.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    "type": "martial_art",
+    "name": "Silat",
+    "extend": { "weapons": [ "afs_energy_saber_on" ] }
+  },
+  {
+    "id": "style_sojutsu",
+    "copy-from": "style_sojutsu",
+    "type": "martial_art",
+    "name": "Sojutsu",
+    "extend": { "weapons": [ "afs_titanium_bat" ] }
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/martialarts.json]: Bats aren't used to jab with the tip, and aren't spears.

File: data/mods/Aftershock/martialarts.json [hunk 6]
```diff
@@ -0,0 +1,79 @@
+[
+  {
+    "id": "style_aikido",
+    "copy-from": "style_aikido",
+    "type": "martial_art",
+    "name": "Aikido",
+    "extend": { "weapons": [ "afs_hydraulic_gauntlet" ] }
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/martialarts.json]: Can you still grab opponents with these? #34208

File: data/mods/Aftershock/martialarts.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    "type": "martial_art",
+    "name": "Fior Di Battaglia",
+    "extend": { "weapons": [ "afs_titanium_bat" ] }
+  },
+  {
+    "id": "style_judo",
+    "copy-from": "style_judo",
+    "type": "martial_art",
+    "name": "Judo",
+    "extend": { "weapons": [ "afs_hydraulic_gauntlet" ] }
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/martialarts.json]: See Aikido comment.

Comment (Maleclypse) [reply data/mods/Aftershock/martialarts.json]: I try and help you and now all my tests are failing ;P.  But yeah I just wandered through and tossed in whatever might possibly fit.  I've removed your requests.

**Pull request merged**